draftjs_exporter/entities.py

```diff
@@ -27,23 +27,12 @@ def render(self, props):
 
 
 class Link:
-    attributes = ['url', 'rel', 'target', 'title']
-
-    @staticmethod
-    def is_valid_attribute(key):
-        # TODO How much do we need to whitelist / blacklist attributes?
-        valid_data_attr = key.startswith('data-') and len(key) > 5 and key.islower()
-        return key in Link.attributes or valid_data_attr
-
     def render(self, props):
```

Comment:
I agree with the whole exporter or nothing.

As for removing the whole file, I reckon we should have entities for everything that ships by default with DraftJS, otherwise you would format something in the editor, but the exporter would just ignore it.